app/printing/printLegacy/get.controller.js

```diff
@@ -0,0 +1,36 @@
+const { logger } = require('../../../common/logging/logger')
+const { removeUrlLevels } = require('../../../common/utils/util')
+const { getOasysSentencePlan } = require('../../../common/data/sentencePlanningApi')
+
+const printLegacySentencePlan = async ({ path, params: { id, planId }, tokens }, res) => {
+  try {
+    const backUrl = `${removeUrlLevels(path, 2)}/plans`
+
+    let legacyPlan
+
+    // get the OASys sentence plan
+    try {
+      legacyPlan = await getOasysSentencePlan(id, planId, tokens)
+    } catch (error) {
+      logger.error(`Could not retrieve OASys sentence plan ${planId} for offender ${id}, error: ${error}`)
+      return res.render('app/error', { error })
+    }
```

Comment:
Do we need an internal catch here?
The error is already logged by the `sentencePlanningApi`.